### DISCUSSION TITLE
Thoughts about introducing telemetry/statistics

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#2706] [Polls]
Hello,

As you may know, UniGetUI currently does not collect any kind of usage data.
However, I have been thinking about introducing some kind of (maybe optional?) usage statistics/telemetry into UniGetUI.

The statistics could serve to:
 - Give me (the developer) a general idea of which features are more used, to focus improvements and development there, be able to identify when a feature is useless, etc.
 - Perhaps create some kind of **package rankings** (most installed packages, etc.) that could show on a new Home Page on UniGetUI (related to https://github.com/marticliment/UniGetUI/issues/2456)

Obviously, the statistics would be 100% anonymous, and the data wouldn't be sold or anything, but could be used to create public, open _databases_, such as the previously-mentioned package rankings.

I am not sure on how to approach this, so I'd like to hear your thoughts.
Make sure to vote on the post and let your ideas on the comments.
Thanks :)

**Comment by Lockszmith-GH** [comment]
There is an active discussion with the wingert project about similar functionality on winget's side.

The product manager there raises some interesting concerns, might be worth checking that out.

I'll update with the link in a bit (currently on my phone)

**Comment by randompersononinternet69** [comment]
Yes, have that feature, but leave it off by default. We don't want this to be like Firefox which respects your privacy, but it enables telemetry on by default which we don't want.
